sdk/communication/Azure.Communication.CallingServer/tests/CallConnection/CallConnectionLiveTests.cs

```diff
@@ -401,38 +351,53 @@ public async Task RunCreateAddCreateAudioRoutingRemoveHangupScenarioTests()
                 Assert.NotNull(addParticipantResult);
                 await WaitForOperationCompletion().ConfigureAwait(false);
 
+                // Add Participant To Default Audio Group
+                var addParticipantToDefaultAudioGroupResult = await AddParticipantToDefaultAudioGroupOperation(callConnection, userId).ConfigureAwait(false);
+                Assert.NotNull(addParticipantToDefaultAudioGroupResult);
+                await WaitForOperationCompletion().ConfigureAwait(false);
+
                 // Creating List And Adding Participant
                 List<CommunicationUserIdentifier> participantList = new List<CommunicationUserIdentifier>();
                 participantList.Add(new CommunicationUserIdentifier(userId));
 
-                //Create Audio Routing Group
-                var createAudioRoutingResult = await CreateAudioRoutingGroupOperation(callConnection, participantList).ConfigureAwait(false);
-                string audioRoutingGroupId = createAudioRoutingResult.AudioRoutingGroupId;
+                //Create Audio Group
+                var createAudioGroupResult = await CreateAudioGroupOperation(callConnection, participantList).ConfigureAwait(false);
+                string audioGroupId = createAudioGroupResult.AudioGroupId;
 
-                // Get Audio Routing Group
-                var getAudioRoutingResult = await GetAudioRoutingGroupOperation(callConnection, audioRoutingGroupId).ConfigureAwait(false);
-                Assert.IsTrue(getAudioRoutingResult.AudioRoutingMode == AudioRoutingMode.Multicast);
-                Assert.NotNull(getAudioRoutingResult);
+                // Get Audio Group
+                var getAudioGroupgResult = await GetAudioGroupOperation(callConnection, audioGroupId).ConfigureAwait(false);
+                Assert.IsTrue(getAudioGroupgResult.AudioRoutingMode == AudioRoutingMode.Multicast);
+                Assert.NotNull(getAudioGroupgResult);
 
                 // Add Another Participant
                 AddParticipantResult addAnotherParticipantResult = await AddParticipantOperation(callConnection, anotherUserId).ConfigureAwait(false);
                 Assert.NotNull(addAnotherParticipantResult);
                 await WaitForOperationCompletion().ConfigureAwait(false);
 
+                // Add Another Participant To Default Audio Group
+                var addAnotherParticipantToDefaultAudioGroupResult = await AddParticipantToDefaultAudioGroupOperation(callConnection, anotherUserId).ConfigureAwait(false);
+                Assert.NotNull(addAnotherParticipantToDefaultAudioGroupResult);
```

Comment:
let us not add this to this test. this test should only handle the renaming of AudioRoutingGroup to AudioGroup for Create.Get.Update,Remove